src/Compilers/CSharp/Portable/Syntax/AnonymousMethodExpressionSyntax.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+// TODO2
```

Comment:
Please link an issue to track. Also a description of what this todo is.

Reply:
Oops. Meant to remove that comment (I use that label for tracking things that I need to consider before submitting a PR). I'm leaving this file to be annotated in a later PR.